src/main/proto/wfa/measurement/reporting/v2alpha/metric.proto

```diff
@@ -128,47 +128,66 @@ message MetricSpec {
   VidSamplingInterval vid_sampling_interval = 5;
 }
 
+// Statistics of a scalar value
+message UnivariateStatistics {
```

Comment:
Regarding the naming, a statistic is a function mapping a set of values to another value, and statistics can include, e.g., mean, std, variance, etc. Here, it seems we only use it for std? If yes, then maybe just call it UnivariateStandardDeviation? But if we will include more statistics, then the current name is better.